app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
@klappradla  In fact, I found one place where this controller is used. I created 2 more test scenarios to simulate a user who is not a student and a student that is not in a team to try update attendance.
Now, I believe that is throwing the right error message.